tests/phpunit/unit-tests/Model/TestModelPDFMetaBoxFailure.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+namespace GFPDF\Model;
+
+use GFPDF\Controller\Controller_PDF;
+use GFPDF\Helper\Helper_Url_Signer;
+use GFPDF\View\View_PDF;
+use WP_UnitTestCase;
+
+/**
+ * @package     Gravity PDF
+ * @copyright   Copyright (c) 2020, Blue Liquid Designs
+ * @license     http://opensource.org/licenses/gpl-2.0.php GNU Public License
+ */
+
+/**
+ * Class TestModelPDFMetaBoxFailure
+ *
+ * @package GFPDF\Model
+ *
+ * @group   pdf
+ */
+class TestModelPDFMetaBoxFailure extends WP_UnitTestCase {
+
+	/**
+	 * The WP Unit Test Set up function
+	 *
+	 * @since 4.0
+	 */
+	public function setUp() {
+		global $gfpdf;
+
+		/* run parent method */
+		parent::setUp();
+
+		/* Setup our test classes */
+		$this->model = new Model_PDF( $gfpdf->gform, $gfpdf->log, $gfpdf->options, $gfpdf->data, $gfpdf->misc, $gfpdf->notices, $gfpdf->templates, new Helper_Url_Signer() );
+		$this->view  = new View_PDF( [], $gfpdf->gform, $gfpdf->log, $gfpdf->options, $gfpdf->data, $gfpdf->misc, $gfpdf->templates );
+
+		$this->controller = new Controller_PDF( $this->model, $this->view, $gfpdf->gform, $gfpdf->log, $gfpdf->misc );
+		$this->controller->init();
```

Comment:
Do we need these if we're just testing the Model?

Reply:
On the metabox registration we dont need it. On the other hand, view_pdf_entry_detail requires it as it will prompt an error "Trying to get property 'view' of non-object". Im thinking moving this method to its parent Test file (test-pdf.php).

Reply:
Ah, okay, copy. Let's leave em in there then.